lib/silo/commands/repo_list.rb

```diff
@@ -36,12 +36,13 @@ def run
           puts "No silos found."
         else
           table = Table.new
-          table.headers 'Name', 'Description', 'Platform', 'Public?'
+          table.headers 'Name', 'Description', 'Platform', 'Public?', 'ID'
           Silo.all.each do |s|
             table.row Paint[s.name, :cyan],
                       Paint[s.description, :green],
                       Paint[s.type.name, :cyan],
-                      s.is_public
+                      s.is_public,
+                      s.id.delete_prefix("flight-silo-").upcase
```

Comment:
This doesn't work, since `@id` isn't accessible outside of `Silo`.

Reply:
Also, does it need to be upcased?

Reply:
Not specifically, just thought it looked better that way in the list.

Reply:
Thinking about it more, I'm not sure we even need the ID to be user-visible. I thought that was just for internal use, to make sure we weren't duplicating unique names.

Reply:
Yeah I thought about that and decided including it wouldn't hurt and would allow users to distinguish between silos (if, for example, a user wants to add an existing silo to their system called `my_silo` but they've already created one called `my_silo`), although in retrospect this doesn't make much of a difference since only one of the silos would be accessible given the syntax of the silo commands uses.

What it DOES help with is if a user has any issues with a silo, they have a method of accessing the correct bucket without having to dig into each individual silo's `cloud_metadata` file.

Reply:
Have just pushed a fix to make `@id` accessible